sql/2022/12/opportunity-score-wp-sites-lcp-without-fetchpriority-high.sql

```diff
@@ -0,0 +1,63 @@
+# HTTP Archive query to get % of WordPress sites not having fetchpriority='high' on LCP image.
+#
+# WPP Research, Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/15
+CREATE TEMP FUNCTION getFetchPriorityAttr(attributes STRING) RETURNS STRING LANGUAGE js AS '''
+try {
+const data = JSON.parse(attributes);
+const fetchpriorityAttr = data.find(attr => attr["name"] === "fetchpriority")
+return fetchpriorityAttr.value
+} catch (e) {
+return "";
+}
+''';
+
+SELECT
+  client,
+  with_fetchpriority_on_lcp,
+  (total_with_lcp-with_fetchpriority_on_lcp) AS without_fetchpriority_on_lcp,
+  total_with_lcp,
+  total_wp_sites,
+  CONCAT(ROUND((total_with_lcp-with_fetchpriority_on_lcp)*100/total_with_lcp, 3),' %') AS opportunity_score,
```

Comment:
Can we shorten this to just `opportunity`? It's not really a score, just "opportunity" is sufficient.